CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,5 @@
+- Fix: set service subservice in logs of redirector and configurations
+- Fix: set new trans and corr or use it if provided in logs
```

Comment:
#211 should be mentioned in one of these?

Reply:
Not sure about 211 will be fixed with this PR

Reply:
Ok, I missinterpret the "related #211" in this PR 👍 

NTC